tests/lib/rules/no-unused-vars.js

```diff
@@ -98,6 +115,21 @@ tester.run('no-unused-vars', rule, {
     {
       code: '<template><div v-for="x in items">{{value | x}}</div></template>',
       errors: ["'x' is defined but never used."]
+    },
+    {
+      code: '<template><div v-for="x in items">{{value}}</div></template>',
+      options: [{ varIgnorePattern: 'ignore$' }],
+      errors: ["'x' is defined but never used."]
+    },
+    {
+      code: '<template><span slot-scope="props"></span></template>',
+      errors: ["'props' is defined but never used."],
+      options: [{ varIgnorePattern: '^ignore' }]
+    },
+    {
+      code: '<template><span><template scope="props"></template></span></template>',
+      errors: ["'props' is defined but never used."],
+      options: [{ varIgnorePattern: '^ignore' }]
```

Comment:
Could you add test cases for suggestions?

https://eslint.org/docs/developer-guide/nodejs-api#testing-suggestions